Common/GPU/D3D11/thin3d_d3d11.cpp

```diff
@@ -1329,6 +1332,7 @@ Framebuffer *D3D11DrawContext::CreateFramebuffer(const FramebufferDesc &desc) {
 void D3D11DrawContext::BindTextures(int start, int count, Texture **textures) {
 	// Collect the resource views from the textures.
 	ID3D11ShaderResourceView *views[8];
```

Comment:
might as well change this array length to MAX_BOUND_TEXTURES since you're here poking around

Reply:
Yep, agreed.  Went ahead and cleaned up some other GL checks in Draw too along the same lines.

-[Unknown]